### PR TITLE
Signup: Adding a help tip above the site preview

### PR DESCRIPTION
**Pull request by shaunandrews**
This PR adds a helpful tip above the preview on the `site-topic` and title steps:

image

Additionally, I've tweaked the `box-shadow` for the device frames using Material guidelines.

To test, head to `/start/onboarding-dev`, choose the Business segment, and work your way to the `site-topic` step. Once you start typing, the help tip and previews should transition into view. The help tip should go away once you venture past the title step.

--

There's probably a better way to handle the display logic. I used CSS to show it based on step classNames, but this logic should probably be handled by the steps themselves.

The delayed rendering of the search suggestions is a little weird, and kind of hides the help text when its transitioning. We might be able to make this feel better, but its probably best served in a separate PR.

### CODE REVIEW

**Comment by fditrapani** [comment]
Copy looks great!

File: client/signup/site-mockup/style.scss [hunk 1]
```diff
@@ -66,13 +71,46 @@
...
+.site-mockup__help-tip {
+	color: var( --color-primary-50 );
+	text-align: center;
+	font-size: 13px;
+	font-weight: 300;
+	margin: 24px 0 -8px 0;
+	transition: all 300ms cubic-bezier(.08,.74,.44,1.07);
+	transition-delay: 200ms;
+
+	p {
```
Comment (fditrapani) [inline client/signup/site-mockup/style.scss]: Looking on mobile, we might need some margins on the sides here:

image

Comment (michaeldcain) [reply client/signup/site-mockup/style.scss]: Addressed in 374aae8.

**Comment by fditrapani** [comment]
🚢

**Pull request merged**